pkg/pubsub/adapter/converters/storage.go

```diff
@@ -44,13 +45,20 @@ const (
 	// TODO find the public google endpoint we should use to point to the schema and avoid hosting it ourselves.
 	//  The link above is tied to the go-client, and it seems not to be a valid json schema.
 	storageSchemaUrl = "https://raw.githubusercontent.com/google/knative-gcp/master/schemas/storage/schema.json"
+
+	Finalize       = "google.storage.object.finalize"
+	Archive        = "google.storage.object.archive"
+	Delete         = "google.storage.object.delete"
+	MetadataUpdate = "google.storage.object.metadataUpdate"
+
+	StorageResourceGroup = "storages.events.cloud.run"
 )
 
 func storageSource(bucket string) string {
 	return fmt.Sprintf("%s/%s", storageSourcePrefix, bucket)
 }
 
-func convertStorage(ctx context.Context, msg *cepubsub.Message, sendMode ModeType) (*cloudevents.Event, error) {
+func convertStorage(ctx context.Context, msg *pubsub.Message, sendMode ModeType) (*cloudevents.Event, error) {
```

Comment:
are you sure `cepubsub` to `pubsub` rename is what should happen here?

Reply:
is still the cepubsub, but with a different alias... is confusing, you are right...
things have moved around in the sdk, that is why we need to different imports...
let me know your preference of aliases.

Reply:
I prefer the alias.

Reply:
left it as is, and changed the other one to cepubsubcontext